src/workers/jobs.js

```diff
@@ -60,6 +64,62 @@ export async function uploadContacts(job) {
   }
 }
 
+
```

Comment:
so we will have an input box, and then this function will pass it through like 79